### PR TITLE
Fix #271636: Translate notenames in piano role editor and status bar

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Resolves 
https://musescore.org/en/node/271636

including the piano tool keyboard tooltips, also for piano roll editor, there including the strings for drumsets too.

### CODE REVIEW

File: mscore/pianotools.cpp [hunk 1]
```diff
@@ -247,8 +247,8 @@ PianoKeyItem::PianoKeyItem(HPiano* _piano, int p)
       _highlighted = false;
       type = -1;
 
-      QString pitchNames[] = {"C", "C#", "D", "D#", "E", "F", "F#", "G", "G#", "A", "A#", "B"};
-      QString text = pitchNames[_pitch % 12] + QString::number((_pitch / 12) - 1);
+      const char* pitchNames[] = {"C", "C♯", "D", "D♯", "E", "F", "F♯", "G", "G♯", "A", "A♯", "B"}; // keep in symc with `valu` in limbscore/utils.cpp
```
Comment (Howard-C) [inline mscore/pianotools.cpp:250]: `symc`?

Comment (Jojo-Schmitz) [reply mscore/pianotools.cpp]: thanks, fixed

**Comment by anatoly-os** [comment]
Changing translatable strings is up to 3.6

**Comment by Jojo-Schmitz** [comment]
There's no new string, it just picks on existing translations!

**Comment by Jojo-Schmitz** [comment]
There's no new or changed translatable string, it just picks on existing translations!

**Comment by Jojo-Schmitz** [comment]
@anatoly-os : please recheck/reconsider: There's no new or changed translatable strings, it just picks on existing translations!

**Comment by anatoly-os** [comment]
@Jojo-Schmitz 
I don't see `InspectorAmbitus` strings on Transifex, but 'Inspector Ambitus`.
I see neither `utils` nor `drumset` strings as well.

What do I miss?

**Comment by Jojo-Schmitz** [comment]
These are not the strings, but their context

**Comment by Jojo-Schmitz** [comment]
These are not the strings, but their context. Causing the translation of the actual (and existing unchanged) string getting **reused** here.
Again: this PR does not add or change any existing string!

**Comment by Jojo-Schmitz** [comment]
Again: while this PR indeed does change some strings (namely `PianoKeyboard::pitchNames[]` and `pitchNames[]`), it does **not** change or add any **translatable** strings, it instead just picks up **existing translations**, so I see no reason to not include it in 3.5.1.

**Pull request merged**